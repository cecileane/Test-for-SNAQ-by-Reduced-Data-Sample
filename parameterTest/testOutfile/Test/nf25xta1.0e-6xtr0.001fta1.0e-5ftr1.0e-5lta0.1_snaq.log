optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 0.1.
rootname for files: nf25xta1.0e-6xtr0.001fta1.0e-5ftr1.0e-5lta0.1_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 89911
seed: 89911 for run 1
Sat Oct  1 17:06:21 2016

 BEGIN SNaQ for run 1, seed 89911 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 66 (failures: 25) with -loglik=128.22643 and ht_min=[0.30771,1.73904,8.8789,0.4599,0.66106]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 19		 19		 1	 |	 0		 0		 --
mvtarget	 18		 18		 2	 |	 0		 0		 --
chdir		 15		 15		 1	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 12		 9		 1	 |	 5		 5		 --
Total		 66		 63		 6	 |	 6		 6		 6
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 128.22642800713652
((3,(5,(6)#H7:::0.6922874647593764):1.7390407012316134):0.4598956827466163,((1,#H7:::0.3077125352406236):8.878895401669935,2):0.6610574110253538,4);
---------------------
seed: 28803 for run 2
Sat Oct  1 17:06:27 2016

 BEGIN SNaQ for run 2, seed 28803 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 72 (failures: 25) with -loglik=216.12729 and ht_min=[0.39847,0.11024,0.0,0.99132,9.93186]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 3		 3		 1	 |	 NA 		 NA 		 NA 
mvorigin	 14		 14		 0	 |	 0		 0		 --
mvtarget	 20		 20		 3	 |	 0		 0		 --
chdir		 10		 10		 0	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 25		 19		 0	 |	 1		 1		 --
Total		 72		 66		 4	 |	 1		 1		 1
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 216.12728821592728
(((6,(5,#H7:::0.3984749470555466):9.931857063788847):0.9913176589100473,1):0.11023645885466546,(4,2):0.0,(3)#H7:::0.6015250529444534);
---------------------

Sat Oct  1 17:06:32 2016
MaxNet is ((3,(5,(6)#H7:::0.6922874647593764):1.7390407012316134):0.4598956827466163,((1,#H7:::0.3077125352406236):8.878895401669935,2):0.6610574110253538,4); 
with -loglik 128.22642800713652
