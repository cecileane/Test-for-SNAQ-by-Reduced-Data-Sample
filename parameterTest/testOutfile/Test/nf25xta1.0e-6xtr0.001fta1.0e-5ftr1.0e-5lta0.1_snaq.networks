((3,(5,(6)#H7:::0.6922874647593764):1.7390407012316134):0.4598956827466163,((1,#H7:::0.3077125352406236):8.878895401669935,2):0.6610574110253538,4);, with -loglik 128.22642800713652 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
((3,(5,(6,#H7:::0.4050429034499267):0.34743517900185494):0.3933862006338333):0.5568394648416153,((1)#H7:::0.5949570965500732,2):8.58375786616996,4);, with -loglik 140.83225132969616
((4)#H7:::0.633374341298252,3,(5,(6,((#H7:::0.366625658701748,2):9.152558741529276,1):1.2277370995801953):0.023311341907189448):0.0753859831081213);, with -loglik 182.34924945897072
((3,(5,(6,((2)#H7:::0.5687051932757293,1):9.119477086992536):0.03267858127140114):0.06525614488192984):1.600958926114663,#H7:::0.4312948067242707,4);, with -loglik 197.1292662979701
((3,(5)#H7:::0.5067255384424842):1.155657429786467,((1,(6,#H7:::0.4932744615575158):5.9463712281169325):0.042680439278330956,2):0.20348765871216076,4);, with -loglik 231.1031413797642
(#H7:::0.22794733424533298,((1,(6,(5,(3)#H7:::0.772052665754667):0.010139073746267964):0.7757237005687135):0.0996296556365564,2):1.0294745851276685,4);, with -loglik 263.84843208281876
