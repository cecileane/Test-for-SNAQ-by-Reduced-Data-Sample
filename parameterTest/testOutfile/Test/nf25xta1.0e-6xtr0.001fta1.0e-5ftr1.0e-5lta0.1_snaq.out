((3,(5,(6)#H7:::0.6922874647593764):1.7390407012316134):0.4598956827466163,((1,#H7:::0.3077125352406236):8.878895401669935,2):0.6610574110253538,4); -Ploglik = 128.22642800713652
 Dendroscope: ((3,(5,(6)#H7):1.7390407012316134):0.4598956827466163,((1,#H7):8.878895401669935,2):0.6610574110253538,4);
 Elapsed time: 10.329807126 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((3,(5,(6)#H7:::0.6922874647593764):1.7390407012316134):0.4598956827466163,((1,#H7:::0.3077125352406236):8.878895401669935,2):0.6610574110253538,4);, with -loglik 128.22642800713652
 (((6,(5,#H7:::0.3984749470555466):9.931857063788847):0.9913176589100473,1):0.11023645885466546,(4,2):0.0,(3)#H7:::0.6015250529444534);, with -loglik 216.12728821592728
-------
